Sources/GreenstandWalletSDK/GreenstandWalletSDK.swift

```diff
@@ -36,6 +37,8 @@ public class GreenstandWalletSDK: GreenstandWalletSDKProtocol {
     }
 
     public func signInWallet(walletName: String, password: String, completion: @escaping (Result<String, Error>) -> Void) {
+        apiService.rootWalletName = walletName
```

Comment:
We dont want to do this here - the root wallet is the greenstand wallet not the users wallet.